assets/src/blocks/HubspotForm/HubspotFormEditor.js

```diff
@@ -0,0 +1,147 @@
+import { Fragment, useEffect } from '@wordpress/element';
+import { RichText } from '@wordpress/block-editor';
+import { Sidebar } from './HubspotFormSidebar';
+import { useToAttribute } from './hooks/useToAttribute';
+import { useBackgroundImage } from './hooks/useBackgroundImage';
+import { getStyleFromClassName } from '../getStyleFromClassName';
+
+const { __ } = wp.i18n;
+
+/**
+ * This component is created as a workaround because the <RichText> doesn't provide
+ * The functionality to set max characters like any content editable element.
+ */
+const MaxLengthHelperComponent = ({ value, maxLength, children, darkTheme = false }) => (
```

Comment:
We'll need this functionality in other blocks as well, so maybe we could make it into a separate component already? 🤔 Ofc actually using it in other blocks should be taken care of in another ticket

Reply:
I completely agree with this but I suggest to merge this implementation as a base one and then continue working on improvements in the future.

Reply:
I see your point, I'm just worried that we'll forget this component is defined here when we start implementing this for other blocks... Anyway, it's not a blocker, so I'll leave it up to you 🙂

Reply:
I agree it should probably be a separate component, but let's do that in a follow up so we don't have to rush it.

Reply:
@GP-Dan-Tovbein can you make a follow up ticket to start capturing all the things we need to work on afterwards in order to put the block out of beta?

Reply:
For sure @comzeradd